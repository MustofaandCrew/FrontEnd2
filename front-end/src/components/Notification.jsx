import React from "react";
import "../assets/css/Notification.css";
import { NavLink } from "react-router-dom";

export default function Notification(props) {
  return (
    <li>
      {props.data &&
        props.data.map((data) => (
          <NavLink key={data.id} className="dropdown-item" to="/dashboardbuyer">
            <div className="d-flex product">
              <div className="product-img">
                <img src={data.Product.ProductImages[0].image} alt="Gambar" />
              </div>
              <div className="product-details">
                <h5>{data.Product.nama}</h5>
                <p>Buyer Offer:</p>
                <p>Rp. {data.harga}</p>
                {data.status === "Diterima" && <h5 className="seller-agree">seller agrees to your offer</h5>}
                {data.status === "Ditolak" && <h5 className="seller-reject">seller rejects to your offer</h5>}
              </div>
            </div>
            <hr />
          </NavLink>
        ))}
    </li>
  );
}
