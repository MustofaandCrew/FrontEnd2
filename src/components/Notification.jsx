import React from "react";

import "../assets/css/Notification.css";

// Dummy Images
// import NewProduct2 from "../assets/images/NewProducts2.jpg";

export default function Notification() {
  return (
    <li>
      <a className="dropdown-item" href="/shoppingcart">
        <div className="d-flex product">
          <div className="product-img">
            <img src="https://i.ebayimg.com/thumbs/images/g/rn8AAOSwIGBc1UqJ/s-l300.jpg" alt="" />
          </div>
          <div className="product-details">
            <h5>Jam Tangan Casio</h5>
            <p>Your Offer:</p>
            <p>Rp. 1.500.000</p>

            <h5 className="seller-agree">seller agrees to your offer</h5>
            {/* <h5 className="seller-reject">seller rejects to your offer</h5> */}
          </div>
        </div>
      </a>
      <hr />
      <a className="dropdown-item" href="/shoppingcart">
        <div className="d-flex product">
          <div className="product-img">
            <img src="https://i.ebayimg.com/thumbs/images/g/rn8AAOSwIGBc1UqJ/s-l300.jpg" alt="" />
          </div>
          <div className="product-details">
            <h5>Jam Tangan Casio</h5>
            <p>Your Offer:</p>
            <p>Rp. 500.000</p>

            {/* <h5 className="seller-agree">seller agrees to your offer</h5> */}
            <h5 className="seller-reject">seller rejects to your offer</h5>
          </div>
        </div>
      </a>
    </li>
  );
}
