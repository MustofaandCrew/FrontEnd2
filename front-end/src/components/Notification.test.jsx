import React from "react";
import renderer from "react-test-renderer";
import Notification from "../components/Notification";


describe("Test suits for Notification", () => {
    it("should match with snapshot", () => {
        const tree = renderer.create(<Notification />).toJSON();
    });
});