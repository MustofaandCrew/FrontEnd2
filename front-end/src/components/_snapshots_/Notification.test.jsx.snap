// Jest Snapshot v1, https://goo.gl/fbAQLP
import Notification from "../components/Notification";

exports["Notification"] = {
    test: function test() {
        var tree = renderer.create(<Notification />).toJSON();
        expect(tree).toMatchSnapshot();
    }
};